README.md

```diff
@@ -31,7 +33,17 @@ Run the following commands:
 
 ### Backend
 
-To help with backend implementation, you can reference the `count` demo feature. 
+To help with backend implementation, you can reference the `count` demo feature.
+
+#### Authentication - JWT
+
+Ensure that you have these packages installed: `pip3 install pyjwt` to ensure your project works well with the authorization system. Navigate to your `backend` folder and create yourself a `.env.development`. You will need to run the following command to generate yourself a secret key: `openssl rand -hex 32`. To confirm that the route works as expected, clear your database and run the test data generated for the `teams.py` test data.

```

Comment:
Can you add this to the `requirements.txt` so we can just get it in the install steps?

Reply:
Should already be added, I froze my dependencies into requirements.txt